ecosystem/sep-0006.md

```diff
@@ -720,6 +744,7 @@ Name | Type | Description
 `withdraw_memo_type` | string | (optional) Memo type for `withdraw_memo`.
 `started_at` | UTC ISO 8601 string | (optional) Start date and time of transaction.
 `completed_at` | UTC ISO 8601 string | (optional) Completion date and time of transaction.
+`expires_at` | UTC ISO 8601 string | (optional) Expiration date for the anchor to receive that payment and proceed with the deposit/withdraw at the agreed quote. This value should be present if and only if the deposit/withdraw was submitted using the `quote_id` parameter. If already expired, the wallet should not allow the user to proceed with the deposit/withraw action.
```

Comment:
Same point about omitting this attribute.

Reply:
Agreed! Removed it in https://github.com/stellar/stellar-protocol/pull/973/commits/169e4a1408ddafe6cd87a2008f4384e3a363403f.